FlightJournal.Web/Views/Flight/Create.cshtml

```diff
@@ -98,7 +98,7 @@
                 <div class="clearfix visible-sm visible-xs"></div>
                 <div class="col-md-4 col-sm-12 visible-sm visible-xs">
                     <div class="btn-group btn-group-lg ">
-                        <input type="submit" value="@_("Create")" class="btn btn-primary" />
+                        <input type="submit" value="@_("Create")" class="btn btn-primary" onclick="gridStaleData()" />
```

Comment:
Jeg tror vi skal forsøge at generere notificatons serverside i stedet for fra js.

Men ok som version 1.0, synes jeg.

Reply:
ok, det kan jeg kigge på videre